### PR TITLE
feat: update decentraland-dapps, add support for languages

### PR DESCRIPTION
**Pull request by cazala**
**IMPORTANT**: This requires https://github.com/decentraland/decentraland-dapps/pull/25 to be merged and release, and the version needs to be updated cos now it's pointing to a release candidate.

This PR bumps decentraland-dapps and removes logic from this dapp that has been moved to the core libraries.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "prettier.singleQuote": true,
   "prettier.semi": false,
   "editor.tabSize": 2,
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "typescript.tsdk": "node_modules/typescript/lib"
```
Comment (NicoSantangelo) [inline .vscode/settings.json]: I don't mind this being here but versioning paths makes me nervous so is it necessary?

File: webapp/src/components/PollDetailPage/PollDetailPage.tsx [hunk 2]
```diff
@@ -299,6 +299,8 @@ export default class PollDetailPage extends React.PureComponent<Props, State> {
                       totalPages={totalPages}
                       firstItem={null}
                       lastItem={null}
+                      nextItem={null}
+                      prevItem={null}
```
Comment (NicoSantangelo) [inline webapp/src/components/PollDetailPage/PollDetailPage.tsx]: I only just catch this. Is it necessary? I would have5 imagined that not adding those props yields the same result

File: .vscode/settings.json (hunk 1 above)
Comment (cazala) [reply .vscode/settings.json]: Resolved as discussed here https://github.com/decentraland/decentraland-dapps/pull/25#discussion_r233849611

File: webapp/src/components/PollDetailPage/PollDetailPage.tsx (hunk 2 above)
Comment (cazala) [reply webapp/src/components/PollDetailPage/PollDetailPage.tsx]: Not adding these lines result in the arrows (next/prev) showing up, but they break on mobile causing horizontal scroll. We can't reduce the number of pages due to limitations from the underlying `<Pagination>` component (sematic-ui), so we decided to get rid of the arrows for now (we should address this at `decentraland-ui` level to fix it the right way).

This was already done in PollsTable, but was missing here, so I added it to keep both paginations consistent.

**Comment by abarmat** [comment]
🥇

**Pull request merged**